core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -803,6 +830,10 @@ public void onShutdown() {
    */
   private static final class GlobalDosFilter extends DoSFilter {
 
+    public GlobalDosFilter() {
```

Comment:
I don't think this constructor is needed.

Reply:
Somehow snuck-in, was only for local dev, good spot.